crates/bevy_render/src/lib.rs

```diff
@@ -236,3 +242,45 @@ fn check_for_render_resource_context(context: Option<Res<Box<dyn RenderResourceC
         );
     }
 }
+
+fn window_icon_changed(
+    mut map: Local<HashMap<WindowId, Handle<Texture>>>,
+    textures: Res<Assets<Texture>>,
+    mut windows: ResMut<Windows>,
+    asset_server: ResMut<AssetServer>,
```

Comment:
```suggestion
    asset_server: Res<AssetServer>,
```
I don't believe asset server needs to be mutable.